src/renderer/types/testSequencer.ts

```diff
@@ -8,12 +8,13 @@ export type LockedContextType = {
   isLocked: boolean;
 };
 
-export type TestTypes = "Python" | "Flojoy" | "Matlab";
+export type TestTypes = "Pytest" | "Python" | "Flojoy" | "Matlab";
```

Comment:
What is the difference between Pytest and just Python here (?)

Reply:
TEST_SET_START is already for the test Union, and the "Pytest" | "Python" are for the different types as demo in slack